Node 0.12 issue with execSync

Chrome2calltree is suffering from execSync issue. https://github.com/mgutz/execSync/issues/38
The solution is probably to change to sync-exec.

I was able to get around the issue by installing an older version of node using nvm.

chrome2calltree 
module.js:338
    throw err;
          ^
Error: Cannot find module './build/Release/shell'
    at Function.Module._resolveFilename (module.js:336:15)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (~/.nvm/versions/v0.12.2/lib/node_modules/chrome2calltree/node_modules/execSync/index.js:30:11)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)

--- a/bin/chrome2calltree.js
+++ b/bin/chrome2calltree.js
@@ -3,7 +3,7 @@ var fs = require("fs");
 
 var argparse = require("argparse");
 var temp = require("temp");
-var execSync = require("execSync");
+var exec = require('sync-exec');
 
 // Clean up temporary files on exit
 temp.track();
@@ -74,8 +74,8 @@ var launchKCacheGrind = function(logpath) {
     var commands = ['qcachegrind', 'kcachegrind'];
     for (var i = 0; i < commands.length; i++) {
         var cmd = commands[i];
-        if (execSync.exec('which ' + cmd).code === 0) {
-            execSync.run(cmd + ' ' + outStream.path);
+        if (exec('which ' + cmd).status === 0) {
+            exec(cmd + ' ' + outStream.path);
             found = true;
             break;
         }

--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
     "argparse": "~0.1.15",
     "lodash": "~2.3.0",
     "underscore.string": "~2.3.3",
-    "execSync": "~1.0.1",
+    "sync-exec": "~0.6.2",
     "temp": "~0.6.0"
   },
   "license": "MIT",

